premake5.lua

```diff
@@ -9,6 +11,11 @@ workspace "Hazel"
 		"Dist"
 	}
 
+	workspace_files 
+	{
+		".editorconfig"
+	}
+
```

Comment:
Overscoped a little by calling it `workspace_files`. I think this can just be an `editorconfig` field since we have no plans to use it for anything else.

Reply:
I agree that workspace files sounds a bit weird. Since it's VS related, I would tend to call it `solution files` ? 0x actualy called them Solution Items, which sounds appropriate as well, since it's not limited to files only...